beacon_node/lighthouse_network/src/behaviour/mod.rs

```diff
@@ -902,7 +902,7 @@ impl<TSpec: EthSpec> NetworkBehaviourEventProcess<RPCMessage<TSpec>> for Behavio
             debug!(
                 self.log,
                 "Ignoring rpc message of disconnecting peer";
-                "peer" => %peer_id
+                "event" => ?event,
```

Comment:
Did you mean to remove the `peer` as well?

Reply:
the peer is included in the event, so yes

Reply:
th event is the peer an its connection plus the msg